README.md

```diff
@@ -101,9 +101,9 @@ Verify the installation by running:
 Once Ruby and JavaScript are installed, install Ruby dependencies by running:
 
     gem install bundle
-    bundle install
```

Comment:
Funnily enough, I think this is wrong (and I know it's been in here a while). I think this needs to be `gem install bundler`.